dask/dataframe/io/io.py

```diff
@@ -524,7 +524,8 @@ def to_records(df):
 
 
 @insert_meta_param_description
-def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
+def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed", 
+                 check_meta_flag=True):
```

Comment:
How about `check_meta` or `verify_meta` instead?  The suffix `_flag` might not be necessary here.

Reply:
I think `check_meta` will bump into the function `check_meta`, but `verify_meta` works for me.